README.rst

```diff
@@ -1,14 +1,75 @@
-A base package for multi-dimensional contiguious and non-contiguious spatially aware arrays.
---------------------------------------------------------------------------------------------
+`NDCube`
+========
 
-.. image:: http://img.shields.io/badge/powered%20by-SunPy-orange.svg?style=flat
-    :target: http://www.sunpy.org
-    :alt: Powered by Sunpy Badge
+|Latest Version| |Build Status| |Build status| |codecov| |Research
+software impact| |Powered by NumFOCUS| |Powered by Sunpy|
 
+.. |Powered by Sunpy| image:: http://img.shields.io/badge/powered%20by-SunPy-orange.svg?style=flat
+   :target: http://www.sunpy.org
+.. |Latest Version| image:: https://img.shields.io/pypi/v/ndcube.svg
+   :target: https://pypi.python.org/pypi/ndcube/
+.. |Build Status| image:: https://secure.travis-ci.org/sunpy/sunpy.svg
+   :target: http://travis-ci.org/sunpy/ndcube
+.. |Build status| image:: https://ci.appveyor.com/api/projects/status/github/sunpy/ndcube?branch=master&svg=true
+   :target: https://ci.appveyor.com/project/sunpy/ndcube
+.. |codecov| image:: https://codecov.io/gh/sunpy/ndcube/branch/master/graph/badge.svg
+   :target: https://codecov.io/gh/sunpy/ndcube
+.. |Powered by NumFOCUS| image:: https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
+   :target: http://numfocus.org
 
+NDcube is a a open-source package for multi-dimensional contiguious and non-contiguious spatially aware arrays.
 
+Installation
+------------
+
+If you want to develop NDcube you will need to install from git.
```

Comment:
@DanRyanIrish we should re-write this now we are about to have a release

Reply:
Just a bit about installing from PyPi, I can do it here if you would like

Reply:
You mean rewrite to explain how to install the stable version?  We could explain how to do both.  Stable first, and then bleeding edge.  How about:

The stable version of ndcube can be installed simply via the anaconda distribution:
```conda install ndcube # Or whatever the full install instructions are```
or via pip:
```pip install ndcube # Or whatever the full install instructions are```

To install the bleeding edge version, you will need to install from git.  The best way to do this is to create a new conda environment and install the git version of ndcube:
```
$ conda config --append channels conda-forge
$ conda create -n ndcube-dev python sunpy hypothesis pytest-mock
$ source activate ndcube-dev
$ git clone https://github.com/sunpy/ndcube.git ndcube-git
$ cd ndcube-git
$ pip install -e .
```
For detailed (general) installation instructions, see...